python/ola/PidStore.py

```diff
@@ -212,7 +204,7 @@ def __str__(self):
     return '%s (0x%04hx)' % (self.name, self.value)
 
   def __hash__(self):
-    return hash((self._name, self._value))
+    return hash(self._name)
```

Comment:
This should be value not name surely?

Reply:
And embarrassingly no tests around that.  Checked and found one other case in uid where hash == wasn't checked (added an additional test around hash != b/c it made it look better though it was actually run via the helper function later)

Reply:
And equally embarrassing, I didn't even consider the fact it wasn't caught by any tests, and we should add one! :man_facepalming: